skimage/measure/fit.py

```diff
@@ -697,6 +697,8 @@ def ransac(data, model_class, min_samples, residual_threshold,
         If RandomState instance, random_state is the random number generator;
         If None, the random number generator is the RandomState instance used
         by `np.random`.
+    init_inliers : [list, tuple of] (N) array of bool or None, optional
```

Comment:
Use "array-like of bool"

Reply:
Oh I just saw what you meant here. Ignore me. =P

Reply:
is `iterate` a better word here?

Reply:
do you suggest to rename all arrays to iterable?

Reply:
No, `iterable of arrays of bool, shape (N,)`

Reply:
Honestly, I'm not sure. I'm having a really hard time wrapping my head around what the dimensionality should be here (just by reading the doc string). I feel like I need an example of having a list of arrays???? Am I interpreting this right?

you say: `list or tuple` but then in the examples you show an array: `init_inliers=np.ones(len(src), dtype=bool)`

I think iterable is the wrong word because then you would be supporting generator expressions.

Maybe `sequence`?

Reply:
I think that it should be `array-like of bool, shape (N,)`